tutorials/embedded/step-0.html.haml

```diff
@@ -0,0 +1,106 @@
+---
+layout: tutorial
+title: Infinispan Embedded Tutorial
+---
+- tutorial_github_url = "https://github.com/infinispan/infinispan-embedded-tutorial"
+- @tutorial_steps = [ "The Weather Application", "Initializing the CacheManager", "Putting stuff in the cache", "Making entries expire", "Configuring the default cache", "Clustering the application", "Listen to changes in the cluster", "Listen to changes in the cache", "Grouping entries together", "A custom externalizer", "Temperature averages with map/reduce", "Declarative configuration" ]
+- tutorial_step = 0
+.row-fluid
+  %h1 Infinispan Embedded Tutorial: The Weather App
+  #columns
+    #leftcol.span3.well
+      = partial( 'tutorialmenu.html.haml', { "tutorial_steps" => @tutorial_steps, "tutorial_step" => tutorial_step } )
+
+    #maincol.span9
+      = partial( 'tutorialnav.html.haml', { "tutorial_steps" => @tutorial_steps, "tutorial_step" => tutorial_step, "tutorial_github_url" => tutorial_github_url, "header" => true } )
+      %p
+        Let's begin by checking out the project. Fire up your terminal and type:
+      %pre
+        %code
+          = preserve do
+            :escaped
+              $ git clone https://github.com/infinispan/infinispan-embedded-tutorial.git
+              $ cd infinispan-embedded-tutorial
+      %p
+        The above commands will create a local copy (clone) of the git repository and change the current directory so that all subsequent commands will run from 
+        the project root directory. We now want to move to "Step 0" in the project:
+      %pre
+        %code
+          = preserve do
+            :escaped
+              $ git checkout -f step-0
+      %p
+        The initial step is just a plain Maven Java project with an implementation of our Weather Application. The application connects to the
+        %a{:href => "http://openweathermap.org"} OpenWeatherMap
+        API to retrieve current weather data for a variety of cities. If you don't have an Internet connection or you are having trouble connecting to 
+        the service, please uncomment the line in the pom.xml file that enables the random weather service. This initial step has no dependencies and uses
+        JDK logging. The following image shows the layout of the project:
+      %img{:src => "../infinispan-embedded-tree.png"}
+      %p
+        Let's compile and run it:
+      %pre
+        %code
```

Comment:
highlightjs detects a keyword here and outputs "package" in bold

%code.nohighlight might be a better option for the command line prompt (even compared to %code.bash which highlights "exec" in the maven cmdline) and the log output (futher down the page in the output section "in" gets highlighted)